<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QDialog" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="Enter">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="welocomeL">
         <property name="text">
          <string>Вас вітає vamos. Щоб продовжити, увійдіть в систему.</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="signInPB">
           <property name="text">
            <string>enter</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="signUpPB">
           <property name="text">
            <string>registr</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="nextPB">
           <property name="text">
            <string>next</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mainWindow">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QPushButton" name="showPlayersPB">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>280</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>show players</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>441</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addPlayerPB">
          <property name="text">
           <string>add player</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addCoachPB">
          <property name="text">
           <string>add coach</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="addPlayer">
      <attribute name="icon">
       <iconset theme="face-wink">
        <normaloff>.</normaloff>.</iconset>
      </attribute>
      <attribute name="title">
       <string>add Player</string>
      </attribute>
      <widget class="QSpinBox" name="ageSB">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="ageL">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>age</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="contractEndDE">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="clubContractDE">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>60</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="contactEndL">
       <property name="geometry">
        <rect>
         <x>158</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>contract end</string>
       </property>
      </widget>
      <widget class="QLabel" name="clubContractL">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>70</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QComboBox" name="footCB">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>120</y>
         <width>72</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Both</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="footL">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>120</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leagueLE">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>180</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="clubLE">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="numberSB">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>170</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="priceLE">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>230</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="addPB">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>270</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>photo</string>
       </property>
      </widget>
      <widget class="QLabel" name="numberL">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>180</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>number</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>110</width>
         <height>116</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="idLE"/>
        </item>
        <item>
         <widget class="QLineEdit" name="nameLE"/>
        </item>
        <item>
         <widget class="QLineEdit" name="surnameLE"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lastNameLE"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="addPlayerToTablePB">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>250</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>add player</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="showPlayer">
      <attribute name="title">
       <string>show players</string>
      </attribute>
      <widget class="QTableView" name="playersTV">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>291</width>
         <height>421</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="idL">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>120</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="surnameL">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>60</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="nameL">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>110</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lastNameL">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>160</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="ageL_2">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>220</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="contractEndL">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>280</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="leagueL">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>50</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="clubL">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>110</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="priceL">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>180</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="numberL_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>240</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="clubContractL_2">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>180</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="prefferedFootL">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>310</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="photoL">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>280</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="searchLE">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>10</y>
         <width>251</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="updatePB">
       <property name="geometry">
        <rect>
         <x>409</x>
         <y>360</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Продовжити контракт</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
